pkg/api/cortex-ruler.go

```diff
@@ -186,11 +203,52 @@ func (c *GettableRuleGroupConfig) validate() error {
 	return nil
 }
 
+// ApiDuration extends model.Duration
+// for handling JSON serialization/deserialization
+type ApiDuration model.Duration
```

Comment:
```suggestion
type Duration model.Duration
```

This ia already part of the api package, I believe it's redundant.

Reply:
sorry, I don't fully understand; why it's redundant?